docs/javascript/components_confirmation.md

```diff
@@ -0,0 +1,100 @@
+# Confirmation - JavaScript API
+
+The purpose of confirmation dialogs is to prevent misclicks and to inform the user of potential consequences of their action.
+A confirmation dialog should always ask a concise question that includes a reference to the object the action is performed upon.
+
+You can exclude extra information or form elements in confirmation dialogs, but these should be kept as compact as possible.
+
+## Example
+
+```ts
+const result = await confirmationFactory()
+  .custom("Do you want a cookie?")
+  .withoutMessage();
+if (result) {
+  // User has confirmed the dialog.
+}
+```
+
+Confirmation dialogs are a special type that use the `role="alertdialog"` attribute and will always include a cancel button.
+The dialog itself will be limited to a width of 500px, the title can wrap into multiple lines and there will be no „X“ button to close the dialog.
+
+## When to Use
+
+Over the past few years the term “Confirmation Fatique” has emerged that describes the issue of having too many confirmation dialogs even there is no real need for them.
+A confirmation dialog should only be displayed when the action requires further inputs, for example, a soft delete that requires a reason, or when the action is destructive.
+
+## Proper Wording
+
+The confirmation question should hint the severity of the action, in particular whether or not it is destructive.
+Destructive actions are those that cannot be undone and either cause a permanent mutation or that cause data loss.
+All questions should be phrased in one or two ways depending on the action.
+
+Destructive action:
+
+> Are you sure you want to delete “Example Object”?
+> (German) Wollen Sie „Beispiel-Objekt” wirklich löschen?
```

Comment:
```suggestion
> (German) Wollen Sie „Beispiel-Objekt“ wirklich löschen?
```
German closing quote is identical to the English opening quote.

English opening: Thick at the bottom, thin at the top.
English closing: Thick at the top, thin at the bottom.
German opening: Thick at the top, thin at the bottom.
German closing: Thick at the bottom, thin at the top.

Thus: Opening and closing are identical under point reflection with a 180° rotation.

Reply:
Thanks, but I’m well aware of this. Mixing both languages sometimes cause me to unconsciously use the wrong keystrokes for the quote marks.
* Option+^ // Shift+Option+^ = „…“
* Option+2 // Shift+Option+2 = “…”